kpops/components/base_components/pipeline_component.py

```diff
@@ -26,6 +26,7 @@ class PipelineComponent(BaseDefaultsComponent):
     from_: FromSection | None = Field(default=None, alias="from")
     app: object | None = None
     to: ToSection | None = None
+    prefix: str | None = None
```

Comment:
should not be nullable

Reply:
should be mandatory ?

Reply:
yes. Please set default as in pipeline_config